server/feeders/restaurantfeeder.js

```diff
@@ -74,8 +74,8 @@ var feedOrionRestaurants = function() {
     .catch(function(err) {
       if (err.statusCode == '404') {
         var address = getAddress(attributes);
-        if (typeof attributes.department !== 'undefined') {
-          fwHeaders['fiware-servicepath'] = '/' + attributes.department;
+        if (typeof attributes.department.value !== 'undefined') {
+          fwHeaders['fiware-servicepath'] = '/' + attributes.department.value;
         }
         setTimeout(function() {
          geocoder.geocode({address: address, country: 'Spain'})
```

Comment:
I've removed it and now I remember why I've used the `setTimeout` function. When doing several requests against the Google maps API, it allows to do 10 requests per second. Sometimes this limit is reached and, we get then the error `[Error: Status is OVER_QUERY_LIMIT. You have exceeded your rate-limit for this API.]`. The only different thing we can do here to avoid the error is, catch it, and then launch the request again if failed; but this can lead into an infinite loop when the 2500 requests per day are reached, as the error returned is the same. So the only thing now it comes to my mind is keep the `setTimeout`... Do you think there is actually a better approach for it? Suggestions are welcome :)